src/demo/models/train.js

```diff
@@ -32,14 +32,22 @@ const footerElements = [
 ];
 
 export const init = () => {
-  const fishData = generateOcean(100);
-  const trainer = new SimpleTrainer();
-  trainer.initializeClassifiersWithoutMobilenet();
+  const state = getState();
+  let fishData = [...state.fishData];
```

Comment:
Does this line do anything?

Reply:
this deep copies `state.fishData` so that we can manipulate the variable `fishData` rather than directly manipulating state